src/components/Tree/utils/removeFragmentsAndEnrichChildren.spec.cy.tsx

```diff
@@ -85,7 +85,7 @@ describe('removeFragmentsAndEnrichChildren', () => {
     });
 
     it('should handle empty fragments', () => {
```

Comment:
Maybe changing the test title to `should handle nullable children`?

Reply:
Actually it needs to be removed as it's already tested in line 80.